app/src/main/java/org/mozilla/focus/search/SearchEngineManager.java

```diff
@@ -63,6 +63,10 @@
     private static final String PREF_KEY_CUSTOM_SEARCH_VERSION = "pref_custom_search_version";
     private static final int CUSTOM_SEARCH_VERSION = 1;
 
+    public enum SearchEngineSource {
+        CUSTOM, BUNDLED
+    }
+
```

Comment:
You are actually never passing an SearchEngineSource object, but instead convert to String immediately. In this case it might be simpler to just use two String constants.